### PR TITLE
Hgvs nomenclature

### PR DESCRIPTION
**Pull request by zfisch**
Determines and adds new hgvs nomenclature to the database. If the new nomenclature is available, it's displayed instead of the old Genomic Coordinate values. If not, the old values are displayed.

### CODE REVIEW

File: pipeline/data_merging/brca_pseudonym_generator.py [hunk 1]
```diff
@@ -85,6 +83,16 @@ def from_field_or_compute(row):
     return s_cdna
 
 
+def compute_genomic_hgvs(cDNA, assemblyMapper):
+    try:
+        genomic_hgvs = assemblyMapper.c_to_g(cDNA)
+        return str(genomic_hgvs)
+    except hgvs.exceptions.HGVSInvalidIntervalError as e:
+        logging.info("HGVSInvalidIntervalError: The given coordinate is outside the bounds of the reference sequence: " +
+                     str(cDNA))
```
Comment (izcram) [inline pipeline/data_merging/brca_pseudonym_generator.py]: I'd catch the `HGVSError`, which is a base class for all sorts of HGVS issues. Also, I'd give some generic message along with the `cDNA` with the message from the actual error, like so:
```
logging.info("Exception during conversion of " + str(cDNA) + " to genomic coordinates: " + str(e)
```

File: website/js/DataTable.js [hunk 2]
```diff
@@ -37,22 +38,31 @@ function setPages({data, count, deletedCount, synonyms, releaseName}, pageLength
...
+        return variantData;
+    },
+    determineGenomicCoordinates: function(variantData) {
+        let field = "Genomic_Coordinate_hg37";
+        let hgvs = variantData.Genomic_HGVS_37;
+        variantData = this.normalizeGenomicCoordinate(field, hgvs, variantData);
+
+        field = "Genomic_Coordinate_hg38";
+        hgvs = variantData.Genomic_HGVS_38;
+        variantData = this.normalizeGenomicCoordinate(field, hgvs, variantData);
```
Comment (izcram) [inline website/js/DataTable.js]: not sure what the intention is, but it seems the `Genomic_Coordinate_hg37` related variables are just overwritten

Comment (zfisch) [reply website/js/DataTable.js]: exactly -- the purpose of this PR is to introduce a new standardized variant nomenclature and use it if it's available. if not, we use the old Genomic_Coordinate_hg38/Genomic_Coordinate_hg37 values.

File: pipeline/data_merging/brca_pseudonym_generator.py (hunk 1 above)
Comment (zfisch) [reply pipeline/data_merging/brca_pseudonym_generator.py]: good call, this is now resolved :)

**Pull request merged**